docs/modules/operation/pages/provisioning/detectors/HostResourceSWRunDetector.adoc

```diff
@@ -1,7 +1,7 @@
 === HostResourceSWRunDetector
 
 This detector test the running state of one or more processes.
-It does this via SNMP by inspecting the _hrSwRunTable_ of the http://www.ietf.org/rfc/rfc2790[HOST-RESOURCES-MIB].
+Detect a process for monitoring using the http://www.ietf.org/rfc/rfc2790[HOST-RESOURCES-MIB].
```

Comment:
Use the http://www.ietf.org/rfc/rfc2790[HOST-RESOURCES-MIB] to detect a process to monitor.